README.md

```diff
@@ -76,123 +76,71 @@ Options](#using-options) below.
-	if err != nil {
-		/* handle error */
-	}
-    // ... etc
-}
-```
-
-[dial-func]: https://pkg.go.dev/github.com/jackc/pgconn#Config
-
-### MySQL
-
-The [Go MySQL driver][mysql] does not provide a stand-alone interface for
-interacting with a database and instead uses `database/sql`. See [the section
-below](#MySQL) on how to use the `database/sql` package with a Cloud SQL MySQL
-instance.
-
-[mysql]: https://github.com/go-sql-driver/mysql
-
-### SQL Server
-
-[Go-mssql][go-mssqldb] does not provide a stand-alone interface for interacting
-with a database and instead uses `database/sql`. See [the section below](#SQL-Server)
-on how to use the `database/sql` package with a Cloud SQL SQL Server instance.
+    // Configure the driver to connect to the database
+    dsn := "user=myuser password=mypass dbname=mydb sslmode=disable"
+    config, err := pgxpool.ParseConfig(dsn)
+    if err != nil {
+        /* handle error */
+    }
 
-[go-mssqldb]: https://github.com/microsoft/go-mssqldb
+    // Create a new dialer with any options
+    d, err := cloudsqlconn.NewDialer(context.Background())
+    if err != nil {
+        /* handle error */
+    }
 
-### Using Options
+    // Tell the driver to use the Cloud SQL Go Connector to create connections
+    config.ConnConfig.DialFunc = func(ctx context.Context, _ string, instance string) (net.Conn, error) {
+        return d.Dial(ctx, "project:region:instance")
+    }
 
-If you need to customize something about the `Dialer`, you can initialize
-directly with `NewDialer`:
+    // Interact with the dirver directly as you normally would
+    conn, err := pgxpool.ConnectConfig(context.Background(), config)
+    if err != nil {
+        /* handle error */
+    }
 
-```go
-myDialer, err := cloudsqlconn.NewDialer(
-    ctx,
-    cloudsqlconn.WithCredentialsFile("key.json"),
-)
-if err != nil {
-    log.Fatalf("unable to initialize dialer: %s", err)
+    // call cleanup when you're done with the database connection
+    cleanup := func() error { return d.Close() }
+    // ... etc
 }
-
-conn, err := myDialer.Dial(ctx, "project:region:instance")
-```
-
-For a full list of customizable behavior, see Option.
-
-### Using DialOptions
-
-If you want to customize things about how the connection is created, use
-`Option`:
-
-```go
-conn, err := myDialer.Dial(
-    ctx,
-    "project:region:instance",
-    cloudsqlconn.WithPrivateIP(),
-)
 ```
 
-You can also use the `WithDefaultDialOptions` Option to specify
-DialOptions to be used by default:
-
-```go
-myDialer, err := cloudsqlconn.NewDialer(
-    ctx,
-    cloudsqlconn.WithDefaultDialOptions(
-        cloudsqlconn.WithPrivateIP(),
-    ),
-)
-```
-
-### Using the dialer with database/sql
-
-Using the dialer directly will expose more configuration options. However, it is
-possible to use the dialer with the `database/sql` package.
+[dial-func]: https://pkg.go.dev/github.com/jackc/pgconn#Config
 
-#### Postgres
+#### Using the dialer with `database/sql`
```

Comment:
Should this be an h5 to be the same as `Using the dialer with pgx`?
```suggestion
##### Using the dialer with `database/sql`
```

Reply:
Yes. Good catch.